src/openrct2/windows/guest_list.c

```diff
@@ -141,7 +141,7 @@ static sint32 _window_guest_list_highlighted_index; // 0x00F1EE10
 static sint32 _window_guest_list_selected_tab;      // 0x00F1EE12
 static sint32 _window_guest_list_selected_filter;   // 0x00F1EE06
 static sint32 _window_guest_list_selected_page;     // 0x00F1EE07
-static uint32 _window_guest_list_selected_view;  // 0x00F1EE13
+static uint32 _window_guest_list_selected_view;		// 0x00F1EE13
```

Comment:
As I said above, I'd be okay with it too, but I think Ted would prefer the file to adhere to the C coding style.